### ISSUE TITLE
[BUG] dataloader in graph classification gives a 2-D label

### ISSUE THREAD

**Issue by StevenGolden1203** [Graph-Learning-Benchmarks/gli#271]
As titled.

**To Reproduce**
```
from dgl.dataloading import GraphDataLoader
from gli.utils import to_dense
import gli
data = gli.dataloading.get_gli_dataset("ogbg-molhiv", "task")
train_data = data[0]
train_loader = GraphDataLoader(train_data, batch_size=128)
for batch, (batched_graph, labels) in enumerate(train_loader):
    print(labels)
    print(labels.shape)
    break
```
The label here is 2D, which fails when counting loss.

![image](https://user-images.githubusercontent.com/84695981/188058702-1abc06f9-8b26-4598-95a8-bea43dc93d32.png)

**Expected behavior**

The label is 1-D.

As comparasion, the dataloader fro DGL will gives a 1D label.

To produce:

```
from dgl.data import GINDataset
from dgl.dataloading import GraphDataLoader
dataset = GINDataset("MUTAG", self_loop=True, degree_as_nlabel=False)
train_loader = GraphDataLoader(dataset, batch_size=128)
for batch, (batched_graph, labels) in enumerate(train_loader):
    print(labels)
    print(labels.shape)
    break
```

![image](https://user-images.githubusercontent.com/84695981/188058407-ac6066b8-9194-4a08-a693-91d6c66f3c62.png)
